src/util/helpers/expressionEvaluators.js

```diff
@@ -77,19 +90,18 @@ const isValid = (attribute): ((?Object) => boolean) => {
  * Helper function for logging data to the console on evaluation error
  *
  * @param message
```

Comment:
@param message
should probably be 
@param errorMessage

Reply:
done